### PR TITLE
Exclude cloned submissions from status checks (WP-684)

### PR DESCRIPTION
**Pull request by vsolovei-smartling**
Cloned submissions were attempting to do a status check. No file is present, and this error is considered unrecoverable, failing the submission.

### CODE REVIEW

File: inc/Smartling/Jobs/LastModifiedCheckJob.php [hunk 1]
```diff
@@ -103,9 +103,7 @@ protected function processFileUriSet(array $submissions, bool $failMissing): arr
                 $lastModified = $this->api->lastModified($submission);
             } catch (SmartlingNetworkException $e) {
                 if ($this->api->isUnrecoverable($e)) {
-                    $submission->setStatus(SubmissionEntity::SUBMISSION_STATUS_FAILED);
-                    $this->submissionManager->storeEntity($submission);
-                    $this->getLogger()->warning("Marking submission {$submission->getId()} failed because of an unrecoverable error in ApiWrapper::lastModified response");
```
Comment (vsolovei-smartling) [inline inc/Smartling/Jobs/LastModifiedCheckJob.php:108]: Logging removed because the code throws exception, which is then logged on an upper level, causing duplicate logging

File: readme.txt [hunk 2]
```diff
@@ -62,6 +62,9 @@ Additional information on the Smartling Connector for WordPress can be found [he
 3. Track translation status within WordPress from the Submissions Board. View overall progress of submitted translation requests as well as resend updated content.
 
 == Changelog ==
+= 2.11.5 =
+* Fixed cloned submissions failing from completed status after time passes
```
Comment (PavelLoparev) [inline readme.txt:66]: I would reword it to something like `Cloned submissions are excluded from status check background job`

**Review by PavelLoparev — APPROVED** [review]
